lib/school.rb

```diff
@@ -0,0 +1,39 @@
+require 'time'
```

Comment:
I'm not seeing where you are using the ruby Time object, so you probably don't need to require it in here :)